corehq/apps/app_manager/templates/app_manager/app_view.html

```diff
@@ -8,7 +8,7 @@
 {% block pre_form_content %}
   <div class="appmanager-page-actions">
     {% include 'app_manager/partials/app_summary_button.html' %}
-    {% include 'app_manager/partials/view_submissions_button_on_white_bg.html' %}
+    {% include 'app_manager/partials/view_submissions_button.html' with btn_style='btn-default' %}
```

Comment:
Just confirming you've tested this works without explicitly passing `form_submit_history_url` here?

Reply:
Yes! tested locally